src/modules/event/event.service.ts

```
@@ -252,6 +253,12 @@ export class EventService {
       whereClauses.push(`ed."metadata"->>'cohortId'='${filters.cohortId}'`);
     }
 
+    if (filters.hasOwnProperty('attendanceMarked')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)